gdx/src/com/badlogic/gdx/Input.java

```diff
@@ -742,6 +742,18 @@ private static void initializeKeyNames () {
 	/** @return whether the menu button is currently being caught */
 	public boolean isCatchMenuKey ();
 
+	/**
+	 * Sets whether the given key on Android should be caught. No effect on other platforms.
+	 * All keys that are not catched may handled by other apps or background processes. For example, media or volume

```

Comment:
All keys that are not catched may **be** handled by other apps or background processes. For example, media or volume